pynest/nest/lib/hl_api_subnets.py

```diff
@@ -277,6 +287,8 @@ def EndSubnet():
 
 
 @check_stack
+@deprecated("", "LayoutNetwork: use Create(<model>, n=<number>) and get a "
+            "GIDCollection")
```

Comment:
Better: "LayoutNetwork is deprecated and will be removed in NEST3. Use Create(<model>, n=<number>) instead."
```Python
@deprecated('LayoutNetwork', 'Create(<model>, n=<number>)')
```